Curricula/src/main/java/com/revature/models/Skill.java

```diff
@@ -6,12 +6,12 @@
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.persistence.JoinColumn;
 import javax.persistence.ManyToOne;
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 
 @Entity
-@JsonIgnoreProperties({"hibernateLazyInitializer", "handler"})// put this on all of your entities
```

Comment:
why was this removed?

Reply:
It was removed because it didnt need to be there. I can add it back in